pkg/util/validation/validate.go

```diff
@@ -100,6 +119,41 @@ func ValidateSample(cfg SampleValidationConfig, userID string, ls []cortexpb.Lab
 	return nil
 }
 
+func ValidateExemplar(userID string, ls []cortexpb.LabelAdapter, e cortexpb.Exemplar) ValidationError {
```

Comment:
It's important to mention the labels retain thing.

```suggestion
// ValidateExemplar returns an error if the exemplar is invalid.
// The returned error may retain the provided series labels.
func ValidateExemplar(userID string, ls []cortexpb.LabelAdapter, e cortexpb.Exemplar) ValidationError {
```